witness/src/Dscp/Witness/SDLock.hs

```diff
@@ -0,0 +1,30 @@
+
+module Dscp.Witness.SDLock
+    ( reading
```

Comment:
Can you please pick up normal names? Like `readSDLock` or `writeSDLock` and `newRWLock`?